packages/Organisms/SiteHeader/index.vue

```diff
@@ -332,9 +344,32 @@ export default {
       }
 
       img {
-        width: $rpl-site-header-logo-width;
+        width: 6.75rem;
```

Comment:
please use a variable here

Reply:
Corrected - updated to use $rpl-site-header-logo-width as cobrand logo width.